lib/mayaUsd/ufe/UsdLight.cpp

```diff
@@ -175,6 +175,11 @@ bool getLightShadowEnable(const UsdPrim& prim)
     const UsdLuxShadowAPI      shadowAPI(prim);
     const PXR_NS::UsdAttribute lightAttribute = shadowAPI.GetShadowEnableAttr();
 
+    if (!lightAttribute) {
+        // If the shadow enable attribute is not created yet, create one here
+        shadowAPI.CreateShadowEnableAttr(VtValue(true));
```

Comment:
Should we default to true here? Does HdStorm render shadows when the attribute doesn't exist?

Reply:
Also, you don't seem to set lightAttribute inside this if, so I'm not sure how the further code works in this case

Reply:
Oh I forgot to assign lightAttribute lol...
So for the default value, I set it as true because I think its natural for a light to cast shadows? I see that's the behavior in Maya (if user created a maya light and duplicated as USD light, it casts shadows as default) so I assume we should do the same for USD light as well... Just dont want the user to be confused.

Reply:
Ok, sounds good. So just make sure to set the lightAttribute, and we'll be good to go.

Reply:
And please do it here and below for the shadow color attribute